packages/zowe-explorer/src/webviews/src/edit-history/components/PersistentToolBar/PersistentClearAllButton.tsx

```diff
@@ -33,7 +33,7 @@ export default function PersistentClearAllButton(): JSXInternal.Element {
   const renderClearAllButton = () => {
     const clearAllText = localize("PersistentClearAllButton.clearAll", "Clear All");
 
-    return selection[type] === "search" || selection[type] === "fileHistory" ? (
+    return selection[type] === "search" || selection[type] === "fileHistory" || selection[type] === "encodingHistory" ? (
```

Comment:
Curious if we should make turn these items into a type (or something we can iterate over) and use it across the `edit-history` components

That way we may be able to simplify this to just  `selection[type] in EditHistory.SelectionType`